habitat_sim/sensors/noise_models/redwood_depth_noise_model.py

```diff
@@ -5,21 +5,27 @@
 # LICENSE file in the root directory of this source tree.
 
 from os import path as osp
+from typing import Union
 
 import attr
 import numba
 import numpy as np
+from numpy import ndarray
 
+try:
```

Comment:
Before, we'd only import Torch lazily, if/when we actually use `RedwoodDepthNoiseModel.simulate`. Now we'll always import it. Any concern here? Is there some chance some env worker that previously didn't import Torch is now importing it? Doesn't torch consume quite a lot of GPU memory on import just due to kernels, etc.?

Reply:
It shouldn't have any side effects until its placed on the GPU so it should be fine. You have to start a torch session before it starts consuming memory, right @erikwijmans ?

Reply:
>You have to start a torch session before it starts consuming memory
my quick experiment shows this to be correct.